tools/cldr-rdf/src/main/resources/org/unicode/cldr/rdf/sparql/wikidata-childToParent.sparql

```diff
@@ -13,14 +13,18 @@ PREFIX bd: <http://www.bigdata.com/rdf#>
 
 
 SELECT DISTINCT ?child ?parent WHERE {
-  ?child p:P279/ps:P279 ?parent.    #child is subclass of parent
```

Comment:
@macchiati  this line referenced 3.3MM rows.  I moved it into the dependent union items